pkg/envoy/xds/node_test.go

```diff
@@ -4,24 +4,22 @@
 package xds
 
 import (
-	. "github.com/cilium/checkmate"
-)
-
-type NodeSuite struct{}
+	"testing"
 
-var _ = Suite(&NodeSuite{})
+	"github.com/stretchr/testify/require"
+)
 
-func (s *NodeSuite) TestEnvoyNodeToIP(c *C) {
+func TestEnvoyNodeToIP(t *testing.T) {
 	var ip string
 	var err error
 
 	ip, err = EnvoyNodeIdToIP("host~127.0.0.1~no-id~localdomain")
-	c.Assert(err, IsNil)
-	c.Check(ip, Equals, "127.0.0.1")
+	require.Nil(t, err)
+	require.Equal(t, "127.0.0.1", ip)
 
 	_, err = EnvoyNodeIdToIP("host~127.0.0.1~localdomain")
-	c.Assert(err, Not(IsNil))
+	require.NotNil(t, err)
```

Comment:
nit: using `require.Error(t, err)` would be more meaningful

Reply:
Thanks, I have updated this line and other places